compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/todo.lower-context-access-destructure-multiple.expect.md

```diff
@@ -15,7 +15,8 @@ function App() {
 ## Code
 
 ```javascript
-import { c as _c } from "react/compiler-runtime"; // @enableLowerContextAccess
+import { useContext_withSelector } from "react-compiler-runtime";
+import { c as _c } from "react/compiler-runtime"; // @lowerContextAccess
 function App() {
   const $ = _c(3);
   const context = useContext(MyContext);
```

Comment:
name doesn't match the name of the generated import

Reply:
yeah because it's not lowered here -- note the todo in the filename

Reply:
i need to keep some state to check if we lower or not and propagate to the result, not sure if it's worth doing? is it an issue to have the unused imports?

Reply:
@jackpope  said this might be an issue so fixed here https://github.com/facebook/react/pull/30628